partcad/src/partcad/globals.py

```
@@ -97,16 +98,26 @@
     return init().render(format, output_dir)
 
 
-def create_package(dst_path=consts.DEFAULT_PACKAGE_CONFIG, private=False):
+def create_package(dst_path=consts.DEFAULT_PACKAGE_CONFIG, config_options={"private": False}):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_package`

Add return type annotation: `bool`

(ANN201)